topics/computational-chemistry/tutorials/cheminformatics/tutorial.md

```diff
@@ -163,6 +163,13 @@ We will generate our compound library by searching ChEMBL for compounds which ha
 
 There are some other tools available, which will not be used in this tutorial, which help to develop a more focused compound library. For example, the 'Natural product likeness calculator' and 'Drug-likeness' tools assign a score to compounds based on how similar they are to typical natural products and drugs respectively, which could then be used to filter the library. If you are interested, you can try testing them out on the library just generated.
 
+> ### {% icon tip %} Tip: Generating a compound library
+> If you try using this tutorial using your own data, you might encounter some issues. Important things to remember:
+> * If you encounter an error, check the SMILES file only has a single column. Additional columns can be removed using the 'Cut' tool.
```

Comment:
at some point we should enhance the chembl tool to take 2 column smiles files as well

Reply:
Yes definitely